dotnetv3/Cognito/scenarios/Cognito_Basics/Cognito_MVPTests/CognitoMethodsTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace Cognito_MVP.Tests
+{
+    public class CognitoMethodsTests
+    {
+        private readonly IConfiguration _configuration;
+        private readonly CognitoMethods _methods;
+        private static readonly AmazonCognitoIdentityProviderClient _client = new AmazonCognitoIdentityProviderClient(RegionEndpoint.USWest2);
+
+        public CognitoMethodsTests()
+        {
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from JSON file.
+                .AddJsonFile("testsettings.local.json",
+                true) // Optionally load local settings.
+            .Build();
+
+            _methods = new CognitoMethods();
+        }
+
+        [Fact]
+        public async Task AdminRespondToAuthChallengeTest_BasMFA_ShouldRaiseError()
```

Comment:
is this supposed to be "Ba**d**MFA_ShouldRaiseError" ?

Reply:
Yes. Fixed.